<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTrip</class>
 <widget class="QDialog" name="NewTrip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(253, 253, 253);
background-image: url(:/pic/background.png);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>99</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>起点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>99</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>途经城市1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>99</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>途经城市2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>99</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>途经城市3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>99</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>途经城市4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>途经城市5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>途经城市6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>途经城市7</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>途经城市8</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>途经城市9</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>终点</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>策略选择</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="timeBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>时间优先</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="priceBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>价格优先</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="mixBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>限时价格优先</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="OK">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>560</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>560</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>141</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="cityLayout"/>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>99</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>时间限制</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>交通工具</string>
   </property>
   <widget class="QRadioButton" name="air">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>飞机</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rail">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>火车</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="road">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>汽车</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pic/transport01.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pic/transport02.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pic/transport03.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="hs">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>460</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>460</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>小时</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="mins">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>460</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>460</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>分钟</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>NewTrip</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>645</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>606</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OK</sender>
   <signal>clicked()</signal>
   <receiver>NewTrip</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>652</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>599</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
